src/Sulu/Bundle/AdminBundle/FormMetadata/StructureLoader.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\FormMetadata;
+
+use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\FormMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\FormMetadataLoaderInterface;
+use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\TypedFormMetadata;
+use Sulu\Component\Content\Metadata\Factory\StructureMetadataFactory;
+use Sulu\Component\Content\Metadata\StructureMetadata;
+use Sulu\Component\Content\Metadata\StructureMetadata as ContentStructureMetadata;
+use Symfony\Component\Config\ConfigCache;
+use Symfony\Component\Config\Resource\FileResource;
+
+class StructureLoader implements FormMetadataLoaderInterface
```

Comment:
When I've read that now, I think the naming of the Loaders is a bit inconsistent... How about `StructureFormMetadataLoader` and `XmlFormMetadataLoader`?